mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -1986,6 +1986,42 @@ static Value foldScalarExtractFromFromElements(ExtractOp extractOp) {
   return fromElementsOp.getElements()[flatIndex];
 }
 
+// If the dynamic operands of `extractOp` or `insertOp` is result of
```

Comment:
[nit] Lets be specific and say exactly which operands this is referring to.
```suggestion
// If the dynamic indices of `extractOp` or `insertOp` is result of
```
Also, this would be more grammatical
```suggestion
// If the dynamic indices of `extractOp` or `insertOp` are result of
// `constantOp`, then fold them.
```